src/Microsoft.Extensions.Logging.Azure.AppServicesDiagnostics/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.0-*",
+  "version": "1.0.0-*",
```

Comment:
We'll need to remove packages from aspnetcore-*-dev feeds that have already been pushed